modules/lib/default.nix

```diff
@@ -105,4 +105,9 @@ in
   # the first string should be the name of the module,
   # followed by the local config attrset
   mkAccentOpt = mkBasicOpt "accent" types.accentOption;
+
+  assertXdgEnabled = config: name: {
```

Comment:
`config` is an argument for this file after #40. it shouldn't need to be provided as an argument